README.md

```diff
@@ -13,6 +13,8 @@ This document attempts to codify that knowledge for people to (re)discover the a
 * Use wait-free data synchronization.
 * Partition data to avoid sharing (and therefore synchronization).
 * Make shared data structures read-only (when possible).
+* Maximise cache hit rate.
+* Inline functions
```

Comment:
The impact of inlining functions on latency not immediately obvious to me. Could you elavorate a bit on what you mean here?

Reply:
I added inline functions because they are a way to avoid the function call overhead. But I guess I was really think about trying to avoid calling functions. Do you think it makes sense to have?

Reply:
Avoiding work is definitely one pattern of low latency programming. Inlining to reduce function call overhead and enable more optimizations is one example of avoiding work. So how about a more high-level bullet point as the pattern (like avoiding work?) and inlining as an example?

Reply:
I agree and have updated the Readme - please feel free to change the wording!